packages/fxa-react/components/Portal/index.tsx

```diff
@@ -5,16 +5,48 @@ import './index.scss';
 
 type PortalProps = {
   id: string;
+  headerId?: string;
+  descId?: string;
```

Comment:
Do these need to be optional if they get defaulted to `''` when `Portal` is instantiated?

Reply:
Yes, it doesn't type check on defaults, but what's passed in.